configure.ac

```diff
@@ -170,6 +170,7 @@ AH_TEMPLATE([HAVE_MALLOC_SIZE],       [Have malloc_size() function])
 AH_TEMPLATE([HAVE_BACKTRACE],         [Have backtrace(3) function])
 AH_TEMPLATE([HAVE_SYSTEMD],           [Have systemd available])
 AH_TEMPLATE([HAVE_ISAL],              [Have Intel ISA-L available])
+AH_TEMPLATE([HAVE_OPENSSL],           [Have OPENSSL available])
```

Comment:
Please remove this if not required. If you want to use this then refer `HAVE_ISAL` in code

Reply:
Removed.